.github/workflows/cicd.yml

```
@@ -240,6 +240,18 @@ jobs:
           labels: |-
             commit-sha=${{ github.sha }}
 
+      - name: "Deploy Tests to Cloud Run"
+        if: ${{ matrix.region == 'europe-west4' }}
```

actionlint (1.7.7)

property "region" is not defined in object type {}

(expression)